### PR TITLE
UI-Design: Config detail. I18n: Modify changed to edit

### PR DESCRIPTION
**Pull request by zyz**
# Description

_1.UI-Design: Config detail. 
  2.I18n: Modify changed to edit_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Config.razor [hunk 1]
```diff
@@ -176,7 +176,7 @@
                                     <MExpansionPanelContent>
                                         @if (configObject.RelationConfigObjectId != 0)
                                         {
-                                            <MCard Style="border:1px solid #E2E7F4;" Class="py-2 mt-4 rounded-5">
+                                            <MCard Style="border:1px solid #E2E7F4; border-radius: 8px !important;" Class="py-2 mt-4">
                                                 <SDataTable FixedHeader
```
Comment (MayueCif) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Config.razor:180]: class rounded-lg

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Config.razor [hunk 2]
```diff
@@ -233,11 +233,11 @@
                                                            Left
                                                            TabChanged="tabText=>HandleTabIndexChanged(tabText, configObject)"
                                                            TabMinWidth="70"
-                                                           ItemsClass="rounded-5"
+                                                           ItemsClass="rounded-2"
                                                            Dense
                                                            Tabs="@(new List<string>{ T("Table"), T("Text") })">
                                                     <SElevationTabItem>
-                                                        <MCard Style="border:1px solid #E2E7F4; padding: 1px;" Class="py-2 mt-4 rounded-5">
+                                                        <MCard Style="border:1px solid #E2E7F4; padding: 1px; border-radius: 8px !important;" Class="py-2 mt-4">
```
Comment (MayueCif) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Config.razor:240]: class rounded-lg

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/ConfigItems.razor [hunk 3]
```diff
@@ -118,6 +119,6 @@
     </MMenu>
     @if (!Disabled)
     {
-        <span class="mr-6 regular3--text btn">|</span>
+        <div class="separator"></div>
```
Comment (MayueCif) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/ConfigItems.razor:122]: MDivider?

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Config.razor.cs [hunk 4]
```diff
@@ -95,13 +95,13 @@ private List<DataTableHeader<ConfigObjectPropertyModel>> Headers
             {
                 var headers = new List<DataTableHeader<ConfigObjectPropertyModel>>()
                 {
-                    new() { Text = T("State"), Value = nameof(ConfigObjectPropertyModel.IsPublished), Width = 170 },
-                    new() { Text = T("Key"), Value = nameof(ConfigObjectPropertyModel.Key), Width = 170 },
-                    new() { Text = T("Value"), Value = nameof(ConfigObjectPropertyModel.Value), Width = 170  },
+                    new() { Text = T("State"), Value = nameof(ConfigObjectPropertyModel.IsPublished) },
```
Comment (MayueCif) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Config.razor.cs:98]: 一个固定宽度都没有吗

**Review by MayueCif — COMMENTED** [review]
@zyz

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Config.razor.cs (hunk 4 above)
Comment (zyz) [reply src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Config.razor.cs:98]: 朱迪设计如此，表格均分

**Review by MayueCif — APPROVED** [review]

**Pull request merged**